src/components/Drawer/EditReportDrawer/EditReportDrawerContent.tsx

```diff
@@ -199,6 +221,17 @@ const EditReportDrawerContent: React.FC = () => {
           </p>
           <StatusValueSelect />
         </div>
+        {/* report description */}
+        <div className="flex flex-row space-x-1 mt-1 items-center">
+          <div className="basis-0.5/12">
+            <Image
+              radius="none"
+              src={poiEditDrawerStatusValue}
+              alt="status value"
+            />
+          </div>
+          <EditDescription />
```

Comment:
Same as above, change the component name

Reply:
I changed the name of the component to StatusDescriptionEdit.